notebooks/community/neo4j/graph_paysim.ipynb

```diff
@@ -1,1138 +1,1121 @@
 {
```

Comment:
<p>Add Workbench link (and modify URLs accordingly) as seen in https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb</p><p> </p><p>This has been tested on Vertex AI Workbench?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/599/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>